Competition2017/src/competition/subsystems/shooter_belt/commands/RunBeltSlowlyCommand.java

```diff
@@ -0,0 +1,27 @@
+package competition.subsystems.shooter_belt.commands;
+
+import competition.subsystems.shooter_belt.ShooterBeltSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class RunBeltSlowlyCommand extends BaseShooterBeltCommand {
+    protected final ShooterBeltSubsystem shooterBeltSubsystem;
+    
+    public RunBeltSlowlyCommand(ShooterBeltSubsystem shooterBeltSubsystem) {
+        this.shooterBeltSubsystem = shooterBeltSubsystem; 
+        this.requires(shooterBeltSubsystem);
+    }
+    
+    @Override
+    public void initialize() {
+        log.info("Initializing");
+    }
+
+    @Override
+    public void execute() {
+        if(shooterBeltSubsystem.isAtSpeed()){
```

Comment:
This still seems odd. Why do you need to check if the belts are at speed? Why not the shooter?